lib/auxlib.c

```diff
@@ -300,6 +302,9 @@ void config_load(int *argc, char ***argv, GOptionEntry *app_entries, const char
 
 out:
 	// default common values, if not configured
+	if (rtpe_common_config_ptr->log_name == NULL)
+		rtpe_common_config_ptr->log_name = g_strdup("rtpengine");
+
```

Comment:
Suggestion to make things more consistent towards the recording daemon: omit this part ...  (1/3)